src/Hazelcast.Net/Metrics/MetricsCompressor.cs

```diff
@@ -161,9 +161,12 @@ private void AppendDescriptor(IMetricDescriptor descriptor)
             if (mask.HasNone(DescriptorMask.Unit))
                 _tempOutput.WriteByte((byte) descriptor.Unit);
 
-            //if (mask.HasNone(DescriptorMask.ExcludedTargets))
-            //    tmpDos.WriteByte(MetricTarget.BitSet(descriptor.ExcludedTargets));
+            // include excludeTargets for compatibility purposes (but it's always zero)
+            if (mask.HasNone(DescriptorMask.ExcludedTargets))
+                _tempOutput.WriteByte(0);
 
+            // include excludeTargets for compatibility purposes (but it's always zero)
+            // (the Python client for instance always writes zero here)
```

Comment:
why do we refer to python client here? how is that relevant?

Reply:
Because this "excludeTargets" thing is generally undocumented and the only way to figure what to do it do spy other clients, so this explains *why* we are sending zero here: because other clients seem to do it and it seems to work.